eng/pipelines/common/xplat-setup.yml

```diff
@@ -12,6 +12,13 @@ jobs:
 - template: ${{ coalesce(parameters.helixQueuesTemplate, parameters.jobTemplate) }}
   parameters:
     variables:
+      - name: _containsCoreClrChange
+        value: $[ dependencies.checkout.outputs['SetPathVars_coreclr.containschange'] ]
```

Comment:
Seems odd that containschange doesn't have word case. Very minor style thing, but kinda funny that someone *could* think this means "contain [snake]" based on typo'd German. 😄

Reply:
:smile: will change it to be camel case.